### PR TITLE
Nixos service fixes

### PR DESCRIPTION
**Pull request by jbgi**

### CODE REVIEW

File: nix/nixos/cardano-node-service.nix [hunk 1]
```diff
@@ -102,14 +100,6 @@ in {
...
       };
-      instanced = mkOption {
-        type = types.bool;
-        default = false;
-        description = ''
-          Whether to enable systemd service instancing.
-          For details see https://fedoramagazine.org/systemd-template-unit-files/
-        '';
-      };
       instances = mkOption {
```
Comment (deepfire) [inline nix/nixos/cardano-node-service.nix:103]: Isn't this also coming along with `instanced`, or is this the other instancing implementation?

**Review by deepfire — APPROVED** [review]
LGTM, thank you @jbgi!

File: nix/nixos/cardano-node-service.nix (hunk 1 above)
Comment (jbgi) [reply nix/nixos/cardano-node-service.nix:103]: no, `instances`` is not implemented through systemd template, just regular nix-generated services.

**Pull request merged**